lib/engine-native/src/engine.rs

```diff
@@ -193,16 +177,40 @@ impl Engine for NativeEngine {
 #[derive(Clone, Copy)]
 pub(crate) enum Linker {
     None,
-    Clang10,
+    // this should be kept in sync with the llvm version required for wasmer-compiler-llvm
```

Comment:
Actually, this does not need to be kept in sync with the compiler-llvm version! I'm not sure what the oldest version that works is, but whatever version the user has installed up to some minimum (that is probably around clang 6) will work fine. Since we know clang-10 works, I'd appreciate it if we could keep support for it as we add clang-11.